docs/triggers_rules.md

```diff
@@ -141,3 +142,6 @@ As an example, create a rule that calls the hello action whenever a location upd
   You see that the hello action received the event payload and returned the expected string.
 
   You can create multiple rules that associate the same trigger with different actions.
+
+The trigger and action that make a rule must be in the same namespace, which is in fact private.
```

Comment:
So we don't have this restriction any more?
If this is fixed I would prefer to remove it then.

So any restrictions now?
trigger can be created on any namespace, and the action can be created on any namespace and package, then the rule can be created with full qualified names to be sure the correct entities are specified?